common/txmgr/txmgr.go

```diff
@@ -239,13 +240,15 @@ func (b *Txm[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) Close() (m
 		}
 		if b.fwdMgr != nil {
 			if err := b.fwdMgr.Close(); err != nil {
-				return errors.Wrap(err, "Txm: failed to stop ForwarderManager")
+				merr = errors.Join(merr, pkgerrors.Wrap(err, "Txm: failed to stop ForwarderManager"))
 			}
 		}
 
 		b.wg.Wait()
 
-		b.txAttemptBuilder.Close()
+		if err := b.txAttemptBuilder.Close(); err != nil {
+			merr = errors.Join(merr, pkgerrors.Wrap(err, "Txm: failed to close TxAttemptBuilder"))
+		}
```

Comment:
@aalu1418 Is there some reason that this need to happen after `Wait()`? If so, could use a comment. If not, would be faster to `Wait` last.